ibridgesgui/__main__.py

```diff
@@ -302,5 +303,11 @@ def main():
     app.exec()
 
 
+def main_deprecated():
+    """Deprecate the ibridges-gui call."""
+    logger = logging.getLogger(THIS_APPLICATION)
+    logger.warning(
+        "'ibridges-gui' will be deprecated. Use 'ibridges gui', needs ibridges 1.5.1 or higher.")
```

Comment:
But this is confusing, if you see this message it means you have `1.5.1` or higher, so there's no point?

Reply:
It can be that you have ibridgesgui 1.5.1 but still run on ibridges 1.5.0. Unless we also fix that in the pyproject toml. 
Either way would be fine for me. @qubixes Do you have a preference?